cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/status/ArmInstanceStatus.java

```diff
@@ -0,0 +1,29 @@
+package com.sequenceiq.cloudbreak.cloud.arm.status;
+
+import com.sequenceiq.cloudbreak.cloud.model.InstanceStatus;
+
+public enum ArmInstanceStatus {
+
+    STARTED("running"),
+    STOPPED("stopped");
+
+    private final String status;
+
+    private ArmInstanceStatus(String status) {
+        this.status = status;
+    }
+
+    public String getStatus() {
+        return status;
+    }
+
+    public static InstanceStatus get(String status) {
+        if (status.equals(STOPPED.getStatus())) {
```

Comment:
The switch-case in ArmStackStatus.java looks nicer, than a multiple if statement.